modules/i2s/i2s_src.c

```diff
@@ -86,8 +86,12 @@ static void *read_thread(void *arg)
 			convert_pcm(st, i, n / 4);
 			i += (n / 4);
 		}
-
-		st->rh(st->sampv, st->sampc, st->arg);
+		
+		struct auframe af = {
```

Comment:
Please move declaration to the top of the function body!

Reply:
And please use auframe_init like for play.